src/main/java/duke/DukeException.java

```diff
@@ -0,0 +1,7 @@
+package duke;
```

Comment:
Perhaps consider organizing exceptions into their own package, since it would help developers easily understand the code base when all classes have been grouped into packages!